composer.json

```diff
@@ -8,15 +8,14 @@
     },
     "require": {
         "php": "^5.6 || ^7.0",
-        "composer-plugin-api": "^1.1"
+        "composer-plugin-api": "^1.1 || ^2.0"
     },
     "require-dev": {
-        "composer/composer": "^1.9",
-        "dealerdirect/phpcodesniffer-composer-installer": "^0.5.0",
+        "composer/composer": "^1.9 || ^2.0@dev",
+        "composer/semver": "^1.5 || ^2.1@dev",
```

Comment:
We should be able to change it - now version 3 is released, so I believe we can remove this requirement completely.

Reply:
Feel free to do that in another PR. I will do nothing until this is merged. There are hundrets of dependency changes every day. Next week might be v4 and we are already merged.
This PR frustrated me at a level I cannot even describe.

Reply:
Done... I hope composer wont use another `@dev` in one of the next commits...